src/std/os/signal.ss

```diff
@@ -22,7 +30,13 @@ package: std/os
   sigaddset
   sigdelset
   sigismember
-)
+  
+  SIG_BLOCK SIG_UNBLOCK SIG_SETMASK)
+
+(cond-expand
+  (openbsd
+    (def SIGRTMIN 1)
+    (def SIGRTMAX 32)))
```

Comment:
that's interesting -- so OpenBSD does not implement real-time signals?

Reply:
I think we should use a different max signal discriminant in this case -- ie dont define `SIGRTMIN` and `SIGRTMAX`, but define a `SIGMAX` instead that is used by the signal handler methods to check validity.